oe_multilingual.post_update.php

```diff
@@ -16,3 +16,15 @@
 function oe_multilingual_post_update_00001_invalidate_containers_cache(): void {
   \Drupal::service('kernel')->invalidateContainer();
 }
+
+/**
+ * Disable English translation.
+ *
+ * Disable English translations to prevent configuration changes to be
+ * stored as locale translations.
+ */
+function oe_multilingual_post_update_00002_disable_english_translation(): void {
```

Comment:
This needs to go in EWCMS. We cannot disable this on existing sites we do not control.